packages/atlas/src/components/Avatar/Avatar.styles.ts

```diff
@@ -173,8 +173,8 @@ export const StyledSvgIllustrativeFileFailed = styled(SvgIllustrativeFileFailed)
   }
 `
 
-export const BadgeContainer = styled.div`
-  position: relative;
+export const BadgeContainer = styled.div<{ 'data-badge': string | number | boolean | undefined }>`
+  position: ${(props) => (props['data-badge'] ? 'relative' : 'contents')};
```

Comment:
"contents" is a display value not a position

Reply:
I think this would work:
```suggestion
  position: relative
  ${(props) => props['data-badge'] && { display: 'contents' }}
```